C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackBotWorker.cs

```diff
@@ -135,16 +279,24 @@ public async Task<object> ReplyWithDialog(object source, object dialogObj)
         /// <param name="update">An object in the form `{id: <id of message to update>, conversation: { id: <channel> }, text: <new text>, card: <array of card objects>}`</param>
         public async Task<object> UpdateMessage(IBotkitMessage update)
```

Comment:
I think that `Task<DeletedResponse>` should be replaced with `Task<UpdateResponse>`

Reply:
I don't know about that, the method _DeleteActivityAsync_ returns void.